<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PanelOrdenesAsignadas</class>
 <widget class="QMainWindow" name="PanelOrdenesAsignadas">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>682</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ordenes Asignadas</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lblTitulo">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>40</y>
      <width>371</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Ordenes Asignadas</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tablaOrdenes">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>130</y>
      <width>401</width>
      <height>271</height>
     </rect>
    </property>
    <property name="columnCount">
     <number>3</number>
    </property>
    <column>
     <property name="text">
      <string>Fecha</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Vehículo</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Estado</string>
     </property>
    </column>
   </widget>
   <widget class="QComboBox" name="cmbNuevoEstado">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>400</y>
      <width>181</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="editable">
     <bool>false</bool>
    </property>
    <property name="currentText">
     <string>Pendiente</string>
    </property>
    <item>
     <property name="text">
      <string>Pendiente</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>En reparación</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Finalizado</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="btnActualizarEstado">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>400</y>
      <width>221</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Actualizar Estado</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnVolver">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>520</y>
      <width>221</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Volver</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
